package/features/steps/lower_requirements.txt

```diff
@@ -1,13 +1,14 @@
 ipython==7.0.0
 fastapi==0.100.0
-fsspec==2021.4
+fsspec==2021.06.01
```

Comment:
can we confirm is shareable viz wont break. I don't have aws/or gcp to test.

Reply:
These are only lower requirements. In general we have `fsspec>=2021.4` , so I think we should be good